gfx/portraits/portrait_modifiers/wc_infusion_appearances.txt

```diff
@@ -0,0 +1,1009 @@
+				value = 0.6
+			}
+		}
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				AND = {
+					has_infusion_level_trigger = { MAGIC = light_void AMOUNT = -2 }
+					NOT = { has_infusion_level_trigger = { MAGIC = light_void AMOUNT = -3 } }
+					NOT = { has_trait = being_void }
+				}
+			}
+		}
+	}
+	
+	being_void = {
+		dna_modifiers = {
+			morph = {
+				mode = add
+				gene = gene_dark_hair
+				template = dark_hair
+				value = 0.15
+			}
+			color = {
+				mode = replace
+				gene = hair_color
+				x = { 0.824 0.902 }
+				y = { 0.824 0.875 }
+			}
+			color = {
+				mode = replace
+				gene = eye_color
+				x = { 0.850 0.920 }
+				y = { 0.358 0.408 }
+			}
+			morph = {
+				mode = add
+				gene = gene_saturated_eyes
+				template = saturated_eyes
+				value = 1
+			}
+			accessory = {
+				mode = add
+				gene = special_eyes
+				template = magic_eyes
+				value = 1
+			}
+			morph = {
+				mode = add
+				gene = gene_albino
+				template = albino_magic
+				value = 1
+			}
+		}
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				OR = {
+					has_infusion_level_trigger = { MAGIC = light_void AMOUNT = -3 }
+					has_trait = being_void
+				}
+			}
+		}
+	}
+}
+
+arcane_appearance = {
+	usage = game
+	
+	level_1 = {
+		dna_modifiers = {
+			color = {
+				mode = replace
+				gene = eye_color
+				x = { 0.650 0.720 }
+				y = { 0.261 0.285 }
+			}
+		}
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				AND = {
+					has_infusion_level_trigger = { MAGIC = arcane_fel AMOUNT = 1 }
+					NOT = { has_infusion_level_trigger = { MAGIC = arcane_fel AMOUNT = 2 } }
+					NOT = { has_trait = being_order }
+				}
+			}
+		}
+	}
+	
+	level_2 = {
+		dna_modifiers = {
+			color = {
+				mode = replace
+				gene = eye_color
+				x = { 0.650 0.720 }
+				y = { 0.300 0.350 }
+			}
+			morph = {
+				mode = add
+				gene = gene_saturated_eyes
+				template = saturated_eyes
+				value = 0.25
+			}
+			accessory = {
+				mode = add
+				gene = special_eyes
+				template = magic_eyes
+				value = 1
+			}
+		}
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				AND = {
+					has_infusion_level_trigger = { MAGIC = arcane_fel AMOUNT = 2 }
+					NOT = { has_infusion_level_trigger = { MAGIC = arcane_fel AMOUNT = 3 } }
+					NOT = { has_trait = being_order }
+				}
+			}
+		}
+	}
+	
+	being_order = {
+		dna_modifiers = {
+			color = {
+				mode = replace
+				gene = eye_color
+				x = { 0.675 0.745 }
+				y = { 0.450 0.500 }
+			}
+			morph = {
+				mode = add
+				gene = gene_saturated_eyes
+				template = saturated_eyes
+				value = 0.9
+			}
+			accessory = {
+				mode = add
+				gene = special_eyes
+				template = magic_eyes
+				value = 1
+			}
+		}
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				OR = {
+					has_infusion_level_trigger = { MAGIC = arcane_fel AMOUNT = 3 }
+					has_trait = being_order
+				}
+			}
+		}
+	}
+}
+
+fel_appearance = { #TODO: Probably needs refactor, seems way too long
```

Comment:
I don't think there's a way to refactor it.

Reply:
;(